### ISSUE TITLE
Blocky Application icon in Debian Buster

### ISSUE THREAD

**Issue by jhclare** [AirVPN/Eddie#113]
The Eddie application icon is very blocky both in the Dock and in Task Switcher. Using Gnome 3.30.2 on Debian Buster. It's been like this for a good while, at least a year.

The 256x256px .PNG icon referred to in the eddie-ui.desktop file looks fine when opened in an image editor.

`Icon=/usr/share/pixmaps/eddie-ui.png`

Dock:
![Dock](https://user-images.githubusercontent.com/29303915/106597511-703f8e80-654e-11eb-8c51-9716ff6fd0c1.png)

Switcher:
![Task switcher](https://user-images.githubusercontent.com/29303915/106597514-72095200-654e-11eb-9ef0-eca2ac77a310.png)

PNG opened in GIMP:
![GIMP](https://user-images.githubusercontent.com/29303915/106597513-7170bb80-654e-11eb-8d40-aed607c20fdc.png)
![Task switcher](https://user-images.githubusercontent.com/29303915/106597514-72095200-654e-11eb-9ef0-eca2ac77a310.png)

Any ideas?

**Comment by AirVPN** [comment]
You are right, but it's related to Mono, this issue will be resolved when we release the 3.x series based on GTK.
